### PR TITLE
Prince Asiedu add

### PR DESCRIPTION
**Pull request by PrinceAsiedu**

### CODE REVIEW

**Comment by PrinceAsiedu** [comment]
Hi @reshamas, kindly review my PR please

**Comment by reshamas** [comment]
@PrinceAsiedu 
This time you submitted the photo.  But you did not submit the contributor info (name, location, etc.)

**Comment by PrinceAsiedu** [comment]
Ok thanks a lot

**Comment by PrinceAsiedu** [comment]
Hi @reshamas, sorry for disturbing you. Very new to this. Thanks for the help too

File: 2022-07_sprint/sprint_parties/contributors.md [hunk 1]
```diff
@@ -450,4 +450,15 @@ Bangalore
...
+<img alt="Image for contributor template" src="../../_static/contributors_2022_07/Prince_Asiedu.jpg" class="rounded-circle">
+
+Prince Asiedu
+
+Accra, Ghana
+
+[{fab}`github`](https://github.com/PrinceAsiedu)
+[{fab}`twitter`](https://twitter.com/PrinceA13827013
+[{fab}`linkedin-in`](https://www.linkedin.com/in/princeoforhasiedu/)
+:::
```
Comment (reshamas) [inline 2022-07_sprint/sprint_parties/contributors.md:464]:
```suggestion
:::
::::
```

File: 2022-07_sprint/sprint_parties/contributors.md [hunk 2]
```diff
@@ -450,4 +450,16 @@ Bangalore
...
+:::{grid-item}
+
+<img alt="Image for contributor template" src="../../_static/contributors_2022_07/Prince_Asiedu.jpg" class="rounded-circle">
+
+Prince Asiedu
+
+Accra, Ghana
+
+[{fab}`github`](https://github.com/PrinceAsiedu)
+[{fab}`twitter`](https://twitter.com/PrinceA13827013
```
Comment (reshamas) [inline 2022-07_sprint/sprint_parties/contributors.md:462]:
```suggestion
[{fab}`twitter`](https://twitter.com/PrinceA13827013)
```

**Comment by reshamas** [comment]
Thanks @PrinceAsiedu.  Your information is now up on the Contributors page.

**Pull request merged**